projects/plugins/super-cache/wp-cache-phase2.php

```diff
@@ -509,13 +509,13 @@ function wpsc_get_accept_header() {
 	static $accept = 'N/A';
 
 	if ( $accept === 'N/A' ) {
-		$json_list = array( 'application/json', 'application/activity+json', 'application/ld+json' );
+		$json_list = apply_filters( 'wpsc_accept_list', array( 'application/json', 'application/activity+json', 'application/ld+json' ) );
 		// phpcs:ignore WordPress.NamingConventions.ValidVariableName.VariableNotSnakeCase
 		// phpcs:ignore WordPress.Security.ValidatedSanitizedInput.MissingUnslash -- $accept is checked and set below.
 		$accept = isset( $_SERVER['HTTP_ACCEPT'] ) ? strtolower( filter_var( $_SERVER['HTTP_ACCEPT'] ) ) : '';
 
 		foreach ( $json_list as $header ) {
-			if ( strpos( $accept, $header ) ) {
+			if ( strpos( $accept, $header ) !== false ) {
```

Comment:
`$accept` is lowercased, but since there's now a filter for `$json_list`, I think `$header` should be lowercased as well, or simply use `stripos`.

Reply:
Thanks, good catch!